build/repo.targets

```diff
@@ -0,0 +1,38 @@
+<Project>
+  <Import Project="common.props" />
+
+  <PropertyGroup>
+      <SiteExtensionProjectDirectory>$(RepositoryRoot)src\Microsoft.AspNetCore.AzureAppServices.TestBundle\</SiteExtensionProjectDirectory>
+      <PublishFeed Condition="$(PublishFeed) == ''">https://dotnet.myget.org/F/aspnetcore-ci-dev/</PublishFeed>
+      <DotnetChannel>master</DotnetChannel>
+      <DotnetVersion>coherent</DotnetVersion>
+  </PropertyGroup>
+
+  <Target Name="BuildSiteExtension">
+
+    <PropertyGroup>
+      <SiteExtensionWorkingDirectory>$(RepositoryRoot).se\</SiteExtensionWorkingDirectory>
+    </PropertyGroup>
+
+    <RemoveDir Directories="$(SiteExtensionWorkingDirectory)" />
+    <Exec Command="cmd /c $(MSBuildThisFileDirectory)\dotnet-install.cmd -Channel $(DotnetChannel) -Version $(DotnetVersion) -Architecture x86 -InstallDir $(SiteExtensionWorkingDirectory) -NoPath" />
+    <MSBuild Projects="$(SiteExtensionProjectDirectory)Microsoft.AspNetCore.AzureAppServices.TestBundle.csproj"
+      Targets="Restore;Pack"
+      Properties="DotnetHomeDirectory=$(SiteExtensionWorkingDirectory)" />
```

Comment:
Add `BuildNumber=$(BuildNumber)` to this list. 

Also, consider putting this into artifacts. `PackageOutputPath=$(ArtifactsDir)siteext\;`